readme.md

```diff
@@ -119,14 +158,24 @@ What is the difference between the two following lines of code?
 @artist.save!
 ```
 
-> Your answer...
+> .save! - will raise an error if not successful.
+  .save - will not raise an error
 
 ## Question 10
 
 Using jQuery, write an AJAX request to `http://tunr.com/artists` that would create a new artist with the name of 'Resin Laying Deer Figurine, Gold', and pop up a box saying "All done!" when complete.
 
 ```js
-// Your code starts here...
+$(".ajax_post").on("click", function(){
+  $.ajax({
+    type: 'POST',
+    data: {artist: {name: "Resin Laying Deer Figurine, Gold"}},
+    dataType: 'json',
+    url: "http://tunr.com/artists"
+  }).done(function(response) {
+    console.log(All done!);
+  })
+})
 
 ```
```

Comment:
Question 11 Is looking for a JS constructor function with a prototype method for an instance of a `instructor` to be able to `receivePresent` a la something like:

```

function Instructor(name){
  this.name = name;
}
Instructor.prototype.receivePresent = function(gift){
  console.log(this.name + " promptly drops the " + gift + " on the floor.");
}

var andy = new Instructor("Andy");
andy.receivePresent("Resin Laying Deer Figurine, Gold");
```